src/js/components/FormField/stories/Select.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, FormField, Select, Grommet } from 'grommet';
+import { allOptions } from './data';
```

Comment:
```suggestion

const allOptions = Array(100)
  .fill()
  .map((_, i) => `option ${i + 1}`);
```
Users have requested no to hide the data in the stories, or add a link to see the data.
In this case, the data is only taking 3 lines, so I think we can embedded it in the file.

Reply:
fixed